Web3privacynowplatform/Briefs/Waku.md

```diff
@@ -0,0 +1,85 @@
+**Basic info**
+
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Project name**  | Waku |
+| **Logotype**  | https://github.com/waku-org/docs.waku.org/blob/develop/static/img/logo_white.svg |
+| **Short project description (1 sentence)** | A family of robust, censorship-resistant communication protocols designed to enable privacy-focused messaging for web3 apps. |
+| **Launch date** | March 2022  |
+| **Is it open source?** | Yes |
+| **Website**  | https://waku.org |
+| **Github**  | https://github.com/waku-org |
+
+**Token**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Do you have a token** | No |
+
+**Socials**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Blog** | https://blog.waku.org/ |
+| **Twitter (X)** | https://x.com/waku_org |
+| **Telegram** | https://t.me/waku_org |
+| **Discord** | https://discord.waku.org |
+| **Other** | https://logos.co |
+
+**Docs**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Docs** | https://docs.waku.org |
+
+**Team**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Is your team anon or public**  | public | 
+| **List core contributors** | https://github.com/orgs/waku-org/people |
```

Comment:
One additional thing - if you look at this without being logged into GH, only `public` contributors will show up. So we should either encourage people to switch their setting or maybe not provide the link here as it only shows me and Richard for people outside of `waku-org`:)

Reply:
Maybe we can skip that for now.